### ISSUE TITLE
Security Questions

### ISSUE THREAD

**Issue by Clickbaitcake** [Terminals-Origin/Terminals#194]
Hello,

My organization are looking to "approve" an RDP manager for our staff.
Can anyone point me to documentation or guidelines on how Terminals stores username, password and domain? Is there any encryption used?

**Comment by jirkapok** [comment]
Yes, security audit was performed in v 2.0, since than all versions are considered secure. All secure sensitive data like user name, domain and passwords are protected, when master password is used. But no FIPS check was done, so we not confirm to be FIPS complient.
